### PR TITLE
Revamp llama.cpp docs

### PR DESCRIPTION
**Pull request by mishig25**
Revamp llama.cpp server docs

### CODE REVIEW

**Review by Vaibhavs10 — APPROVED** [review]
Nice! Could we mention here that they can run any model from the Hub the same way here as long as they pass the `hf-repo` and `hf-file`

**Comment by gary149** [comment]
I would also add it at the beginning of the main README?

**Review by nsarrazin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,6 +30,71 @@ A chat interface using open source models, eg OpenAssistant or Llama. It is a Sv
 7. [Deploying to a HF Space](#deploying-to-a-hf-space)
 8. [Building](#building)
 
+## Quickstart Locally
```
Comment (gary149) [inline README.md:34]:
```suggestion
## Quickstart
```

File: README.md [hunk 2]
```diff
@@ -30,6 +30,71 @@ A chat interface using open source models, eg OpenAssistant or Llama. It is a Sv
...
+
+You can quickly have a locally running chat-ui & LLM text-generation server thanks to chat-ui's [llama.cpp server support](https://huggingface.co/docs/chat-ui/configuration/models/providers/llamacpp).
```
Comment (gary149) [inline README.md:36]:
```suggestion
You can quickly start a locally running chat-ui & LLM text-generation server thanks to chat-ui's [llama.cpp server support](https://huggingface.co/docs/chat-ui/configuration/models/providers/llamacpp).
```

File: README.md [hunk 3]
```diff
@@ -20,15 +20,79 @@ load_balancing_strategy: random
...
+2. [Setup](#setup)
+3. [Launch](#launch)
+4. [Web Search](#web-search)
+5. [Text Embedding Models](#text-embedding-models)
+6. [Extra parameters](#extra-parameters)
+7. [Common issues](#common-issues)
+8. [Deploying to a HF Space](#deploying-to-a-hf-space)
+9. [Building](#building)
+
+## Quickstart
```
Comment (gary149) [inline README.md:34]: maybe add a link to llama.cpp server docs somewhere. (it's quite nice)

Comment (mishig25) [reply README.md:34]: I have link here to hf docs https://github.com/huggingface/chat-ui/blob/50edca76928a1fc6d3b131f07ae4315165d18b7f/README.md#L36 and this hf docs contain link to llama server readme https://github.com/huggingface/chat-ui/blob/50edca76928a1fc6d3b131f07ae4315165d18b7f/docs/source/configuration/models/providers/llamacpp.md#L21-L22

**Comment by mishig25** [comment]
@Vaibhavs10 https://github.com/huggingface/chat-ui/pull/1214/commits/50edca76928a1fc6d3b131f07ae4315165d18b7f

**Comment by mishig25** [comment]
lets make changes in the follow up PR if needed

**Pull request merged**